src/pds/api_client/test/integration/test_collections_of_bundle.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from pds.api_client import Configuration
+from pds.api_client import ApiClient
+from pds.api_client.api.product_references_api import ProductReferencesApi
+
+
+class CollectionsOfBundleTestCase(unittest.TestCase):
+    def setUp(self):
+        # create an instance of the API class
+        configuration = Configuration()
+        configuration.host = 'http://localhost:8080'
+        api_client = ApiClient(configuration)
+        self.product_reference = ProductReferencesApi(api_client)
+
+    def test_collections_of_a_bundle_default(self):
+
+        results = self.product_reference.product_members(
+            'urn:nasa:pds:mars2020.spice::3.0',
+            fields=['ops:Data_File_Info.ops:file_ref']
+        )
+        for collection in results.data:
+            urls = collection.properties['ops:Data_File_Info.ops:file_ref']
+            for url in urls:
+                print(url)
+
+    def test_all_collections_of_a_bundle_as_deep_archive_does(self):
+
+        def get_collections(bundle_lidvid):
```

Comment:
Hi @nutjob4life ,

This is the piece of code which tested the pagination as you do in deep-archive.
I will add a reference to here in the deep-archive ticket. Unfortunatly, I missed to commit the previous version of this code, so it will not be as easy for you to see the changes.